src/transformers/generation/logits_process.py

```diff
@@ -1345,6 +1348,45 @@ class WhisperTimeStampLogitsProcessor(LogitsProcessor):
                 max_initial_timestamp_index (`int`, *optional*, defaults to 1):
                     Used to set the maximum value of the initial timestamp. This is used to prevent the model from
                     predicting timestamps that are too far in the future.
+
+    Examples:
+    ``` python
+    >>> import torch
+    >>> from transformers import AutoProcessor, WhisperForConditionalGeneration,GenerationConfig
+    >>> from datasets import load_dataset
+
+    >>> processor = AutoProcessor.from_pretrained("openai/whisper-tiny.en")
+    >>> model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-tiny.en")
+    >>> ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+    >>> inputs = processor(ds[4]["audio"]["array"], return_tensors="pt")
+    >>> input_features = inputs.input_features
+
+    >>> #Change EOS:
+    >>> #This allows the user to select a specific token to terminate the sequence on, in this case it's the word poem(21247)
+    >>> model.generation_config.eos_token_id = 21247
+    >>> generated_ids = model.generate(inputs=input_features,return_timestamps=True)
+    >>> transcription = processor.batch_decode(generated_ids, skip_special_tokens=True)[0]
```

Comment:
```suggestion
    >>> transcription = processor.batch_decode(generated_ids, skip_special_tokens=False)[0]
```
Let's have one example with timestamps no?

Reply:
TIL 👍

Reply:
@jprivera44 let's a) limit the number of examples (one with and another without timestamps is enough); b) use Arthur's suggestion here :)